values-schema.yaml

```diff
@@ -258,7 +258,15 @@ definitions:
         title: Resource group
         description: An Azure resource group.
         $ref: '#/definitions/wordCharacterPattern'
-
+  backupTtl:
```

Comment:
how about moving backupTtl and backupTtl under the backup section?

Reply:
done